postgresql-plugin/src/main/java/io/cdap/plugin/postgres/PostgresSink.java

```diff
@@ -94,6 +99,20 @@ protected FieldsValidator getFieldsValidator() {
     return new PostgresFieldsValidator();
   }
 
+  @Override
+  protected LineageRecorder getLineageRecorder(BatchSinkContext context) {
+    // dbtype, host, port, db from the connection string
+    // table is the reference name
+    String fqn = DBUtils.constructFQN(postgresSinkConfig.getJdbcPluginName(),
+                                      postgresSinkConfig.getConnection().getHost(),
+                                      postgresSinkConfig.getConnection().getPort(),
+                                      postgresSinkConfig.getConnection().getDatabase(),
+                                      context.getInputSchema().getDisplayName(),
```

Comment:
This is not the correct schame name. It is the name of CDAP schema, we need to pass Database Schema here.
In sinks, we do have a config named as `databaseSchema` but that can be nullable.
In source, we might need to parse the import query.